Jenkinsfile.deploy.new

```
@@ -1,12 +1,16 @@
 @Library('jenkins-pipeline-shared-libraries')_
 
+def deployProperties = [:]
```

as discussed in https://github.com/kiegroup/kogito-cloud-operator/pull/420/files#r450517728, it should be

```suggestion
deployProperties = [:]
```

to have global scope